src/transformers/generation/utils.py

```diff
@@ -1315,6 +1313,7 @@ def generate(
         self,
         inputs: Optional[torch.Tensor] = None,
         generation_config: Optional[GenerationConfig] = None,
+        cache_config: Optional[CacheConfig] = CacheConfig(),
```

Comment:
I'd rather make this one part of the generation config, to avoid adding arguments to generate :)

Reply:
Okay, should be similar to watermarking then